chrome/content/zotero/xpcom/utilities.js

```diff
@@ -1527,14 +1484,14 @@ Zotero.Utilities = {
 	 */
 	"itemToCSLJSON":function(zoteroItem) {
 		if (zoteroItem instanceof Zotero.Item) {
-			zoteroItem = zoteroItem.toArray();
+			zoteroItem = Zotero.Utilities.Internal.itemToExportFormat(zoteroItem);
```

Comment:
With a bare attachment and an empty creators array, citeproc-js will render the title and URL of an attachment. That looks an awful mess, mostly, but I think it should be allowed to render.

Bare items have been showing through in the WP plugins (they are screened out in quickFormat, but they show in the Classic view), so they will be present in documents, and a refresh will attempt to retrieve them from Zotero. If the function returns null, citeproc-js could either throw, or render a "missing data" cite (not sure what it would do currently), but either scenario would leave the user without a means of backtracking to the bare item, which they might want to do in order to identify the source and replace the bare item with a proper reference.